test/dotnet-add-package.Tests/dotnet-add-package.Tests.csproj

```diff
@@ -22,9 +22,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(CLI_TestPlatform_Version)" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0-beta4-build1194" />
-    <PackageReference Include="xunit" Version="2.2.0-beta4-build3444" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
```

Comment:
Seems like maybe we should put these versions in a property so they only need to be updated in one place.

Reply:
We should, but there's some confusion around stable bits we want to ref and bits bundled in the product so I wanted to discuss how we want to factor the props. My goal here was to eliminate warnings. There's much less harm in hard coded stable bits that work vs hard coded preview builds.